src/components/v5/common/ActionSidebar/partials/BatchPaymentsTable/hooks.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useCallback, useMemo } from 'react';
+import { createColumnHelper, ColumnDef } from '@tanstack/react-table';
+
+import { formatText } from '~utils/intl';
+import { BatchPaymentsTableModel } from './types';
+import { TableWithMeatballMenuProps } from '~v5/common/TableWithMeatballMenu/types';
+import TokenSelect from '../TokenSelect';
+import UserSelect from '../UserSelect';
+import AmountField from '../AmountField';
+
+export const useBatchPaymentsTableColumns = (
+  name: string,
+): ColumnDef<BatchPaymentsTableModel, string>[] => {
+  const columnHelper = useMemo(
+    () => createColumnHelper<BatchPaymentsTableModel>(),
+    [],
+  );
+
+  const columns: ColumnDef<BatchPaymentsTableModel, string>[] = useMemo(
+    () => [
+      columnHelper.display({
+        id: 'recipient',
+        header: () => formatText({ id: 'table.row.recipient' }),
+        cell: ({ row }) => (
+          <UserSelect key={row.id} name={`${name}.${row.index}.recipient`} />
```

Comment:
I don't think you need fields here. Looking at the designs these columns will just display data right?